daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -139,6 +141,25 @@ APP ID     APP PORT  AGE  CREATED
 nodeapp    3000      16h  2020-07-29 17:16.22
 ```
 
+### Enabling high-availability in an existing Dapr deployment
+
+Enabling HA mode in an existing Dapr deployment requires additional steps.
+
+If you simply run `helm upgrade dapr --set global.ha.enabled=true ...`, this command will fail on upgrading Dapr placement stateful set.
```

Comment:
This section currently reads like a troubleshooting section. It should start with the correct steps that need to be taken and only then address possible failure if a more "naive" approach is taken.